### PR TITLE
Toolkit Landing Page

### PR DESCRIPTION
**Pull request by mkanoria**
@christineNg1028 
## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)

## Implementation description
* 

## Steps to test
1.

## What should reviewers focus on?
* 

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by NirmalHegde** [comment]
The overall look is really nice. Some comments:

- Some of the cards are inconsistent and have weird white lines where they should be solid black

**Comment by NirmalHegde** [comment]
Overall, looks really nice. Some comments:

- There is a weird white line in the card where there shouldn't be
image

- The image inside the card is stretch
image

- Image is blurry
image

File: pages/index.tsx [hunk 1]
```diff
@@ -1,8 +1,366 @@
...
+                                    image="https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/close-up-of-flower-blooming-outdoors-royalty-free-image-739387273-1544039749.jpg"
+                                    progressValue={10}
+                                />
+                            </Grid>
+                            <br />
+                            <br />
+                            <Heading fontSize={24}>Skills</Heading>
+                            <br />
+                            <Grid templateColumns="repeat(3, 1fr)" gap={6}>
+                                <UserToolCard
```
[the remark this answers is not in the document]
Comment (christineNg1028) [reply pages/index.tsx:158]: yep, map will be used once we have a list of all of the tools that will be available

File: pages/index.tsx [hunk 2]
```diff
@@ -1,8 +1,366 @@
...
+                                Lorem ipsum dolor sit amet, consectetur
+                                adipiscing elit, sed do eiusmod tempor
+                                incididunt ut labore et dolore magna aliqua. Ut
+                                enim ad minim veniam, quis nostrud exercitation
+                                ullamco laboris nisi ut aliquip ex ea commodo
+                                consequat. Duis aute irure dolor in
+                                reprehenderit in voluptate velit esse cillum
+                                dolore eu fugiat nulla pariatur. Excepteur sint
+                                occaecat cupidatat non proident, sunt in culpa
+                                qui officia deserunt mollit anim id est laborum.
```
[the remark this answers is not in the document]
Comment (christineNg1028) [reply pages/index.tsx:61]: aye. this is just placeholder for now but once we get the actual text it should be in a var 👍

**Review by NirmalHegde — APPROVED** [review]
Looks good! Just make sure it passes build

**Comment by ChinemeremChigbo** [comment]
Looks good. Please make sure to fix the failing checks:

**Review by ChinemeremChigbo — APPROVED** [review]
Looks good. Please make sure to fix the failing checks eg.
image
If you need help fixing those, let me know!

**Pull request merged**